packages/terra-form-validation/src/terra-dev-site/doc/form-validation/ValidationComponents.2.doc.mdx

```diff
@@ -0,0 +1,34 @@
+import { Badge } from 'terra-form-validation/package.json?dev-site-package';
+
+import DefaultFormValidation from '../example/DefaultFormValidation?dev-site-example';
+import FormSubmitRadioField from '../example/FormSubmitRadioField?dev-site-package-example';
+import FormSubmitCheckboxField from '../example/FormSubmitCheckboxField?dev-site-package-example';
+import FormSubmitDatePicker from '../example/FormSubmitDatePicker?dev-site-package-example';
+import FormSubmitDateInput from '../example/FormSubmitDateInput?dev-site-package-example';
+
+<Badge />
+
+# Component Setup
+
+## InputFields and TextareaFields
+
+Setting up InputFields and TextareaFields is very simple. Encapsulate the desired field inside of a react-final-form Field component like down below. Ensure that the render function contains input and meta in it's props hash, and that the appropriate elements from the meta prop is passed into error and isInvalid.
+
+## CheckboxField and RadioFields
+
+For CheckboxField and RadioField, it's recommended to use the field component provided by terra-form-checkbox and terra-form-radio. They contain props for rendering the error message in a consistent manner.
+
+In the Form render function, ensure that you have the errors prop set in the prop hash. Then set the error and isInvalid states based on the information from that hash.
+
+For each individual Radio or Checkbox, encapsulate that component inside of a react-final-form Field element, using the same name and validate function for each control you wish for the field to contain. **Note:** For having checkboxes that have multiple values, be sure to attach the [] to the name of the Field.
```

Comment:
nit: Lets move this "Note" to a new line.

Reply:
Moved e52f755